demonstrations/tutorial_coherent_vqls.py

```diff
@@ -496,18 +496,22 @@ def prepare_and_sample(weights):
     variational_block(weights)
 
     # We assume that the system is measured in the computational basis.
-    # If we label each basis state with a decimal integer j = 0, 1, ... 2 ** n_qubits - 1,
-    # this is equivalent to a measurement of the following diagonal observable.
-    basis_obs = qml.Hermitian(np.diag(range(2 ** n_qubits)), wires=range(n_qubits))
-
-    return qml.sample(basis_obs)
+    # Therefore, sampling the device will give us a value of 0 or 1 for each qubit (n_qubits)
```

Comment:
```suggestion
    # Therefore, sampling from the device will give us a value of 0 or 1 for each qubit (n_qubits)
```

I think this is more correct :)